src/main.js

```diff
@@ -96,10 +96,31 @@ export default class LevelSelector extends UICorePlugin {
   configureLevelsLabels() {
     if (this.core.options.levelSelectorConfig === undefined) return
 
-    for (var levelId in (this.core.options.levelSelectorConfig.labels || {})) {
-      levelId = parseInt(levelId, 10)
-      var thereIsLevel = !!this.findLevelBy(levelId)
-      thereIsLevel && this.changeLevelLabelBy(levelId, this.core.options.levelSelectorConfig.labels[levelId])
+    var labelCallback = this.core.options.levelSelectorConfig.labelCallback
+    if(labelCallback && typeof labelCallback !== 'function')
```

Comment:
I have now updated my repository to follow the convention, should I add another pull-request, or will the brackets be fixed in one of your commits?